src/solvers/flattening/boolbv_bitwise.cpp

```diff
@@ -18,7 +18,7 @@ bvt boolbvt::convert_bitwise(const exprt &expr)
   if(expr.id()==ID_bitnot)
   {
     DATA_INVARIANT(expr.operands().size() == 1, "bitnot takes one operand");
-    const exprt &op0=expr.op0();
+    const exprt &op0 = to_bitnot_expr(expr).op();
     const bvt &op_bv = convert_bv(op0, width);
```

Comment:
The `DATA_INVARIANT` isn't necessary anymore, and for consistency it would be nice to rename `op0` to `op` here.